ocaml/forkexecd/test/fe_test.ml

```diff
@@ -68,8 +68,24 @@ let shuffle x =
   done ;
   Array.to_list arr
 
+let fd_list () =
+  let pid = Unix.getpid () in
+  let path = Printf.sprintf "/proc/%d/fd" pid in
+  List.filter (* get rid of the fd used to read the directory *)
+    (fun x ->
+      try
+        ignore (Unix.readlink (Filename.concat path x)) ;
+        true
+      with _ -> false
+    )
+    (Array.to_list (Sys.readdir path))
+
+let fd_count () = List.length (fd_list ())
```

Comment:
Could use `Array.length` here.

Reply:
On second thought: it's probably ok as long as lists are short. I had overlooked that `fd_list` filters out FDs that fail `readlink` and so the result is shorter than the `readdir` result.

Reply:
Understood. Done.